edge_bench_1/edge_ren.rs

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Please don't create a separate folder for benches and weights, they should go directly into the pallet's crate.